icu4c/source/common/locid.cpp

```diff
@@ -38,6 +38,7 @@
 #include "unicode/strenum.h"
 #include "unicode/stringpiece.h"
 #include "unicode/uloc.h"
+#include "cmemory.h"
```

Comment:
This is redundant, locid.cpp already includes cmemory.h about 5 lines after this